content/applications/websites/ecommerce/getting_started/catalog.rst

```diff
@@ -1,37 +1,99 @@
-==================================
-How to customize my catalog page 
-==================================
+============================
+Organize the product catalog
+============================
 
-Product Catalog
-===============
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently. The smoother the navigation in the catalog, the better the shopping
+experience, and the higher the user satisfaction.
 
-All your published items show up in your catalog page (or *Shop* page).
+Categorize the product catalog
+==============================
 
-.. image:: catalog/shop.png
+In Odoo, there is a **specific category model** for your eCommerce. This category, which can be
+found on the product form, in the :guilabel:`eCommerce` tab, is a many-to-many field. This allows a
+single product to be added in multiple eCommerce categories, and therefore show up in multiple
+categories on your website.
```

Comment:
```suggestion
In Odoo, there is a **specific category model** for your eCommerce. This category, which can be
found on the product form in the :guilabel:`eCommerce` tab, is a many-to-many field. This allows a
single product to be added to multiple eCommerce categories and appear in multiple ones on your website.
```

Reply:
Changed it up a bit, replaced your "ones" suggestion w/ "categories" to make sure there's no misunderstanding.